spec/support/gold_master_urls.yml

```diff
@@ -63,15 +63,15 @@ json:
     name: get_map_base_data
     version: v2
     v3_ready: true
-    v3_url: /api/v3/contexts/1/map_groups
+    v3_url: /api/v3/contexts/1/map_layers
```

Comment:
?

Reply:
This endpoint was renamed at some point, but the gold master definitions were not updated so I couldn't run the tests to check for any regressions in this PR. I still find those tests useful, so I thought better include an unrelated change here than forget about it again :)

Reply:
ah ok